tests/resource_test.js

```diff
@@ -0,0 +1,65 @@
+/// <reference path="./steps.d.ts" />
+Feature('Resource test');
+
+var FirstLogin = false;
+var loginCookies = [];
+const pdfName = `Dummy PDF ${Math.random() * 1000}`;
+const collectionName = `Dummy Collection ${Math.random() * 1000}`;
+
+Before((I) => {
+    I.login('admin', 'password');
+});
+
+Scenario('test upload a PDF file and verify the upload process', (I) => {
+    I.seeInCurrentUrl('#dashboard');
+    I.click('Library'); I.wait(3);
+    I.seeInCurrentUrl('#resources');
+
+    I.click('Add New Resource'); I.wait(3);
+    I.seeInCurrentUrl('#resource/add');
+    I.see('New Resources');
+
+    I.see('Collection');
+    I.click('Add New'); I.wait(3);
+    I.fillField('CollectionName', collectionName);
+    I.fillField('Description', 'It is a dummy collection');
+    I.fillField('Year', '2017');
+    I.wait(3);
+    I.click('Save'); I.wait(3);
+    I.seeInPopup('Collection saved successfully.');
+    I.acceptPopup(); I.wait(3);
+    I.wait(3);
+    
+    I.fillField('Title', pdfName);
```

Comment:
Each page of the bell app should be a [pageobject](http://codecept.io/pageobjects/), then you should be able to write

`publication_po.add('John Dou'', '2017', .....)`

In the page object class/file you should make the locator text fields like in the codeceptjs example so it can be changed in one place.